### ISSUE TITLE
Flows could not be created  when using symbols from external library

### ISSUE THREAD

**Issue by Myrronth** [abynim/UserFlows.sketchplugin#125]
As soon as I add a symbol from an external library that is linked by Abstract user flows could not be created or updated. The error message as shown in the console.app is:

```
TypeError: null is not an object (evaluating 'destinationArtboard.parentPage')
line: 903
sourceURL: /Users/myrronth/Library/Application Support/com.bohemiancoding.sketch3/Plugins/UserFlows.sketchplugin/Contents/Sketch/script.js
column: 55
```

 This is not the case when I use a symbol from the "Apple iOS UI" or "iOS UI Design" library shipped with sketch.

**Comment by abynim** [comment]
Thanks for reporting this one @Myrronth. It should be fixed in 2.1.3

**Comment by Myrronth** [comment]
Works now, thanks!
